client-internal/src/test/resources/terracotta/10/tc-config-big.xml

```diff
@@ -40,16 +40,16 @@
 
   <servers>
     <server host="localhost" name="Server1">
-      <tsa-port>9410</tsa-port>
-      <tsa-group-port>9430</tsa-group-port>
+<!--      <tsa-port>9410</tsa-port>-->
+<!--      <tsa-group-port>9430</tsa-group-port>-->
```

Comment:
Removed all port assignments in all XML configs to let the port be assigned by the port allcoator.